telethon/client/auth.py

```diff
@@ -153,13 +153,13 @@ async def _start(
                     warnings.warn(
                         'the session already had an authorized user so it did '
                         'not login to the bot account using the provided '
-                        'bot_token (it may not be using the user you expect)'
+                        f'bot_token (it is using ID {me.id})'
```

Comment:
I think we need to make it explicit we're talking about the user ID, but I'm not sure if this wording is any better:

```suggestion
                        f'bot_token (old session user ID is {me.id})'
```

Reply:
old session could imply that it's in the "old session", i.e. the session on disk. How about `now using ID {me.id}`?

Reply:
Well, the old session (session from the disk) is the one still being used (new token is ignored because it's already logged in).

Reply:
Yeah that's a good point. Probably best to log the ID of both